### DISCUSSION TITLE
Resume X ATS SCORING

### DISCUSSION THREAD

**Discussion by Aryan9901** [recodehive/resume-pitch#89] [Q&A] [answered]
Hello sir, My name is Aryan Gupta and i am pursing btech from computer science engineering. I want to ask you that when I am going to apply for a company sometime my resume got rejected every time 

I want to also ask that beside all this tech I know what else I can learn or what type of projects I should make 

[Aryan_Gupta.pdf](https://github.com/user-attachments/files/17005713/Aryan_Gupta.pdf)

[Aryan Gupta Ats friendly.pdf](https://github.com/user-attachments/files/17005715/Aryan.Gupta.Ats.friendly.pdf)

Sir here are my two resume I used interchangeably

My linkedin profile is 
https://www.linkedin.com/in/agaryan

My portfolio is 
https://www.aryang.vercel.app

**Comment by sanjay-kv** [comment, marked as answer]
I didnt see any topmate request for this 
could you check that again: 
if you wanna book again: https://topmate.io/sanjaykv/1047284/pay
use the same freecode you used before.
